packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/java/integrationsteps.md

```diff
@@ -1,3 +1,101 @@
 ## Integration steps
 
-The Java SDK integration steps for this use case are coming soon. See the [Okta Identity Engine Java SDK usage guide](https://github.com/okta/okta-idx-java#usage-guide) for more information.
+### Step 1: User launches sign-in page
+
+Build a sign-in page for your app that captures both the username and password.
```

Comment:
form, not page?

Reply:
This one still needs to be fixed in the "Sign in with password and email factors" use case page.

Reply:
I updated the alt-text in "Sign in with password and phone factor" use case. I will correct the "Sign in with password and email factors" in another branch (there are a few things I will need to correct in that use case).